### PR TITLE
:bug: Allow pets to be bought with gems

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
Interestingly, this is just a UI fix.

### CODE REVIEW

File: app/templates/play/modal/item-details-view.jade [hunk 1]
```diff
@@ -41,8 +41,9 @@ if item && !item.owned
   if item.unequippable
     button.btn.big-font.disabled.unequippable(data-i18n="inventory."+item.get('heroClass').toLowerCase()+"_only")
   else
-    button#selected-item-unlock-button.btn.big-font(data-item-id=item.id, class=(item.get('subscriber') && !me.isPremium() ? "subscribe-button" : "unlock-button"))
-      if item.get('subscriber') && !me.isPremium()
+    - var showSubscribe = item.get('subscriber') && !me.isPremium() && !me.isStudent() && !me.showHeroAndInventoryModalsToStudents()
```
Comment (differentmatt) [inline app/templates/play/modal/item-details-view.jade]: Curious about the showHeroAndInventoryModalsToStudents check here.  Seems irrelevant at this point, since we're already showing items to whomever is here, and just deciding whether they see an unsubscribe button.  I think we only need an `!me.isStudent()` check.

Comment (AndrewJakubowicz) [reply app/templates/play/modal/item-details-view.jade]: Thanks for the review. Will reduce the check to just the `!me.isStudent()`.

**Comment by AndrewJakubowicz** [comment]
Removed misleading check.
